x-pack/plugins/security_solution/public/management/common/routing.ts

```diff
@@ -132,6 +133,13 @@ export const getPolicyTrustedAppsPath = (policyId: string, search?: string) => {
   })}${appendSearch(search)}`;
 };
 
+export const getPolicyEventFiltersPath = (policyId: string, search?: string) => {
```

Comment:
an advice based on my similar work: make this second `search` parameter to be a partial location object.